src/pages/bosses/boss/index.js

```diff
@@ -0,0 +1,272 @@
+                                path={mdiEmoticonDevil}
+                                size={1.5}
+                                className="icon-with-text"
+                            />
+                        </h1>
+                        {bossJsonData &&
+                            <span className="wiki-link-wrapper">
+                                <a href={bossJsonData.wikiLink}>
+                                    {t('Wiki')}
+                                </a>
+                            </span>
+                        }
+                        {bossJsonData &&
+                            <p className='boss-details'>{bossJsonData.details}</p>
+                        }
+                    </div>
+                    <div className="icon-and-link-wrapper">
+                        <img
+                            alt={bossData.name}
+                            className={'item-image'}
+                            loading="lazy"
+                            src={`https://assets.tarkov.dev/${bossNameLower}.jpg`}
+                        />
+                    </div>
+                </div>
+                <h2 className='item-h2' key={'boss-stats-header'}>{t('Boss Stats')}
+                    <Icon
+                        path={mdiPoll}
+                        size={1.5}
+                        className="icon-with-text"
+                    />
+                </h2>
+                <PropertyList properties={bossProperties} />
+                <p>Key:</p>
+                <ul>
+                    <li>Behavior: The general AI behavior of the given boss</li>
+                    <li>Health: Total boss health</li>
+                    <li>Spawn Chance: A percentage chance that the boss spawns on a given map</li>
+                </ul>
+
+                {bossJsonData &&
+                    <h2 className='item-h2' key={'boss-loot-header'}>{t('Boss Loot')}
+                        <Icon
+                            path={mdiDiamondStone}
+                            size={1.5}
+                            className="icon-with-text"
+                        />
+                    </h2>
+
+                }
+                {bossJsonData &&
+                    <p className='boss-details' key={'boss-loot'}>{bossJsonData.loot}</p>
+                }
+
+                <h2 className='item-h2' key={'boss-spawn-table-header'}>{t('Spawn Locations')}
+                    <Icon
+                        path={mdiMapLegend}
+                        size={1.5}
+                        className="icon-with-text"
+                    />
+                </h2>
+                <ul>
+                    <li>Map: The name of the map which the boss can spawn on</li>
+                    <li>Spawn Location: The exact location on the given map which the boss can spawn</li>
+                    <li>Chance: If the "Spawn Chance" is activated for the map, this is the chance that the boss will spawn at a given location on that map</li>
```

Comment:
I would hedge the language here a bit and say it's the "estimated" chance that a boss spawns in a given location and that the estimates may not be accurate.